### PR TITLE
Remove windows support under gpAux.

### PR DESCRIPTION
**Pull request by BaiShaoqi**
Confirmed with GPClient Team, the windows support under gpAux is not
used in master and 6X_STABLE

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: gpAux/Makefile [hunk 1]
```diff
@@ -170,10 +156,8 @@ ifneq (false, ${PG_LANG})
 ifeq ($(BLD_ARCH), sles11_x86_64)
 CONFIGFLAGS+= --with-python
 else
-ifneq ($(BLD_ARCH),win32)
 # GPDB_91_MERGE_FIXME: Our MinGW environment doesn't currently include Perl and Python.
```
Comment (bradfordboyle) [inline gpAux/Makefile]: w/ @d Without the conditional, we're not sure if this comment makes sense.

File: gpAux/Makefile [hunk 2]
```diff
@@ -459,30 +422,24 @@ CLIENTSINSTLOC_LIB_PWARE=$(CLIENTSINSTLOC)/lib/pware
...
 DOC_TYPE=Unix
 SCRIPT=.sh
-else
-DOC_TYPE=Win
-SCRIPT=.bat
-endif
 
 # yes, the doubled gpload lines are correct
-# this gives win32, win64 just gpload.py, as they want, and wrappers on the unix platforms
+# this gives unix platforms just gpload.py, as they want
```
Comment (bradfordboyle) [inline gpAux/Makefile]: w/ @d: We're not sure that this comment is needed anymore and the double gpload line should be

```sh
...
gpload
gpload.py
...
```

The variable `$(PY_EXT)` is only defined in "Makefile.windows"; for Linux it defaults to empty.

File: gpAux/releng/releng.mk [hunk 3]
```diff
@@ -86,13 +86,11 @@ sync_tools: opt_write_test
 	-Divyrepo.host=$(IVYREPO_HOST) -Divyrepo.realm="$(IVYREPO_REALM)" \
 	-Divyrepo.user=$(IVYREPO_USER) -Divyrepo.passwd="$(IVYREPO_PASSWD)" -quiet resolve);
 
-ifeq "$(findstring aix,$(BLD_ARCH))" ""
```
Comment (bradfordboyle) [inline gpAux/releng/releng.mk]: w/ @d: We think we still need this conditional. We will probably still be calling `sync_tools` for aix7 but we don't want to download orca when building for aix7.

Comment (BaiShaoqi) [reply gpAux/releng/releng.mk]: bring it back

File: gpAux/Makefile (hunk 1 above)
Comment (BaiShaoqi) [reply gpAux/Makefile]: Delete the comment.

File: gpAux/Makefile (hunk 2 above)
Comment (BaiShaoqi) [reply gpAux/Makefile]: Nice catch

**Pull request merged**